sphinx/builders/_epub_base.py

```diff
@@ -1,10 +1,9 @@
 # -*- coding: utf-8 -*-
 """
-    sphinx.builders.epub
-    ~~~~~~~~~~~~~~~~~~~~
+    sphinx.builders._epub_base
```

Comment:
You might refer to docutils' package (`docutils.writer._html_base`). Surely its name represents the module is private enough. But, personally, I prefer more formal name to modules like `sphinx.builders.epub.common`.
What do you think?

I know this is bikeshed. so please rename only if you agreed.